.github/workflows/regression_test.yml

```diff
@@ -63,17 +63,17 @@ jobs:
             runs-on: linux.g5.12xlarge.nvidia.gpu
             torch-spec: 'torch==2.3.0'
             gpu-arch-type: "cuda"
-            gpu-arch-version: "12.1"
+            gpu-arch-version: "12.6"
           - name: CUDA 2.4
             runs-on: linux.g5.12xlarge.nvidia.gpu
             torch-spec: 'torch==2.4.0'
             gpu-arch-type: "cuda"
-            gpu-arch-version: "12.1"
+            gpu-arch-version: "12.6"
           - name: CUDA 2.5.1
             runs-on: linux.g5.12xlarge.nvidia.gpu
             torch-spec: 'torch==2.5.1 --index-url https://download.pytorch.org/whl/cu121'
```

Comment:
this is not updated to cu126

Reply:
CI error seems to be caused by test infra docker:

docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running prestart hook #0: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy'
nvidia-container-cli: requirement error: unsatisfied condition: cuda>=12.6, please update your driver to a newer version, or use an earlier cuda container: unknown